certified-connectors/ActiveCampaign/README.md

```diff
@@ -0,0 +1,23 @@
+
+# ActiveCampaign
```

Comment:
@mmchome please include all required sections for the readme file. Please include all required sections. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).

Reply:
done